ci/tests/puppeteer/run.sh

```diff
@@ -270,22 +351,30 @@ if [[ "${RERUN}" != "true" ]]; then
       printgreen "Running initialization script: ${script}"
       chmod +x "${script}"
       eval "${script}"
+      if [[ $? -ne 0 ]]; then
+        echo "Initialization script [${script}] failed."
+        exit 1
+      fi
     done
 
-    runArgs=$(jq -j '.jvmArgs // empty' < "${config}")
+    echo "Getting run args"
+    runArgs=$(jq -j '.jvmArgs // empty' "${config}")
     runArgs="${runArgs//\$\{PWD\}/${PWD}}"
     runArgs="${runArgs} -Xms512m -Xmx2048m -Xss128m -server"
     [ -n "${runArgs}" ] && echo -e "JVM runtime arguments: [${runArgs}]"
 
-    properties=$(jq -j '.properties // empty | join(" ")' < "${config}")
+    echo "Getting properties"
```

Comment:
I'd prefer if you took these out. They just seem like noise to me as trace statements, and don't convey much.